apps/browser/qml/pages/components/Overlay.qml

```diff
@@ -76,11 +76,12 @@ Shared.Background {
         overlayAnimator.showOverlay(action === PageStackAction.Immediate)
     }
 
-    function startPage() {
+    function startPage(action) {
         searchField.enteringNewTabUrl = true
         _showUrlEntry = true
+        _overlayHeight = Qt.binding(function () { return overlayAnimator._fullHeight })
         searchField.resetUrl("")
-        overlay.animator.updateState("startPage", PageStackAction.Immediate)
+        overlayAnimator.showStartPage(action !== PageStackAction.Animated)
```

Comment:
I wasn't quite sure if I should really do this the other way around (`action === PageStackAction.Immediate`) like in that `enterNewTabUrl()` above and then change all calls to `startPage()`. Anyway, feel free to comment.

Reply:
Looks like what I was expecting.